src/type.ts

```diff
@@ -1,26 +1,25 @@
-import { Geometry } from 'geojson';
+import { Feature, FeatureCollection, Geometry } from 'geojson';
 
-export interface Feature {
-    type: string;
-    id: string;
-    geometry: Geometry;
-    properties : FeatureProperties;
-}
+export type OsmGoFeatureCollection = FeatureCollection<Geometry, FeatureProperties>;
+export type OsmGoFeature = Feature<Geometry, FeatureProperties>;
 
-
-interface FeatureProperties{
+export interface FeatureProperties{
     hexColor: string;
     icon: string;
-    id: string;
+    id: number;
     marker: string
     meta: MetaData
     primaryTag: PrimaryTag
     tags: any;
     type: string;
     way_geometry?: Geometry;
-    usedByWays? : boolean
+    usedByWays? : boolean;
+    originalData: OsmGoFeature | null;
+    changeType?: OsmGoChangeType;
```

Comment:
I've added these two properties since they're used in various places in the code. Again please have a look and see if this is okay or not.